src/BTCPayServer.Lightning.All/LightningClientFactory.cs

```diff
@@ -70,8 +71,14 @@ public ILightningClient Create(LightningConnectionString connectionString)
                     AllowInsecure = connectionString.AllowInsecure,
                 }, HttpClient), Network);
             }
+            else if (connectionString.ConnectionType == LightningConnectionType.Eclair)
+            {
+                return new EclairLightningClient(connectionString.BaseUri, connectionString.Password, Network,
+                    new RPCClient(connectionString.BitcoinAuth, connectionString.BitcoinHost, Network));
```

Comment:
Pass the HttpClient to the RPCClient and ElcairLightningClient (can be null)

Reply:
`RPCClient` does not take an `HttpClient`